reference/volumes.html.markerb

```diff
@@ -14,14 +14,18 @@ A Fly Volume is a slice of an NVMe drive on the physical server your Fly App run
 ## Volume considerations
 
 * **A volume belongs to one app**: Every Fly Volume belongs to a [Fly App](/docs/reference/apps/) and you can't share a volume between apps.
-* **A volume exists in one region**: A volume exists in only one region. Only a Machine running in the same region can access it.
+* **A volume exists on one server**: Each volume exists on one server in a single region. It is not network storage.
 * **A volume can attach to one Machine**: You need to run as many volumes as there are Machines. There's a one-to-one mapping between Machines and volumes. A Machine can only mount one volume at a time and a volume can be attached to only one Machine.
 * **Develop your app to handle replication**: Volumes are independent of one another; Fly.io does not automatically replicate data among the volumes on an app, so if you need the volumes to sync up, then your app has to make that happen.
 * **Create redundancy in your primary region**: If your app needs a volume to function, and the NVMe drive hosting your volume fails, then that instance of your app goes down. There's no way around that. You can run multiple Machines with volumes in your app's primary region to mitigate hardware failures.
 * **Create and store backups**: If you only have a single copy of your data on a single volume, and that drive fails, then the data is lost. Fly.io takes daily snapshots and retains them for 5 days, but the snapshots shouldn't be your primary backup method.
 
 Explore other options for data storage in [Databases & Storage](/docs/database-storage-guides/).
 
+## Volume attachment
+
+Fly Volumes and Fly Machines are meant to be paired together, but they are not necessarily always found in pairs. A Fly Volume can be created without a Fly Machine, or a Machine can be destroyed without destroying its volume. In these cases, the volume that’s left is called an "unattached" volume. A Fly Machine that does not require a volume will never attach itself to one. Fly Machines that do require volumes will *always* be attached to one, and any method of creating a new Fly Machine will always create a matching Fly Volume if there is not an unattached one available.
+
 ## Volume redundancy
```

Comment:
I think that's something that should be said, but it should go in some place about scaling & availability in general, not Volumes specifically.

Actually, I'm not sure how much about redundancy needs to be in this Volumes reference at all. I get the sense that this was written in the Nomad era, when Allocs floated between hosts while Volumes, distinctly, did not. Now that Machines are just as pinned to specific servers as Volumes, it would be better to have a general document about scaling, redundancy, and availability, rather than trying to pack it into Volumes.

Reply:
We have this (hasn't been updated for a bit though): https://fly.io/docs/reference/app-availability/

I think we want to keep mentioning redundancy when we talk about volumes because of the risk of single host failures, right?

Reply:
Update to include what you'd said.